compiler/src/scala/quoted/runtime/impl/QuotesImpl.scala

```diff
@@ -2140,7 +2140,7 @@ class QuotesImpl private (using val ctx: Context) extends Quotes, QuoteUnpickler
 
     given MethodTypeMethods: MethodTypeMethods with
       extension (self: MethodType)
-        def isErased: Boolean = self.isErasedMethod
+        def isErased: Boolean = self.hasErasedParams
         def isImplicit: Boolean = self.isImplicitMethod
         def param(idx: Int): TypeRepr = self.newParamRef(idx)
```

Comment:
I assume we know that the parameter is erased because this type is annotated.  We should have a test for this.

This should also be the case for `TermParamClauseMethods.params`.

Reply:
How do I write such test? Is there an example for this?